cmd/kubelet/app/options/options.go

```diff
@@ -36,8 +37,20 @@ import (
 const (
 	defaultRootDir             = "/var/lib/kubelet"
 	experimentalFlannelOverlay = false
+
+	defaultPodInfraContainerImageName    = "gcr.io/google_containers/pause"
+	defaultPodInfraContainerImageVersion = "2.0"
```

Comment:
Should we give a flag to override this image config? Other people have wanted the ability to specify custom images. Its ok to not add the flag in this PR...